openedx/core/djangoapps/django_comment_common/tests.py

```diff
@@ -20,7 +18,7 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
 
-new_contract('basestring', six.string_types[0])
+new_contract('basestring', (str,)[0])
```

Comment:
I think we can change this to
```suggestion
new_contract('basestring', str)
```